src/octopeer-github/content/ContentController.ts

```diff
@@ -171,6 +183,28 @@ class ContentController {
     }
 
     /**
+     * Makes sure that START_WATCHING_PR and STOP_WATCHING_PR are logged to the database.
+     */
+    private hookFocusAndBlurToDom(database: DatabaseAdaptable) {
```

Comment:
Wouldn't it be more logical to put this function before `unhookFocusAndBlurToDom`?